### ISSUE TITLE
Please tell me how to install latest version

### ISSUE THREAD

**Issue by ronved** [jimmejardine/qiqqa-open-source#223]
I have downloaded v80 and v82 .exe files. But I get this, for both versions, when I try to install.​

Windows protected your PC
Microsoft Defender SmartScreen prevented an unrecognized app from starting. Running this app might put your PC at risk.

App: 
setup-v82.0.7357.40407.exe 
Publisher:  
Unknown publisher

**Comment by jordanharp** [comment]
You should be able to click More Info and tell it to install anyway

**Comment by ronved** [comment]
I am through installing and I thought it would ask me to login but it asks for esc, backup/restore. What should i do
(I previously tried v79 and both account and guest library. I also have web account)

**Comment by jordanharp** [comment]
(1) Pressing esc will launch Qiqqa.
(2) Qiqqa open source does not use web accounts. You'll want to use the old
Qiqqa to sync a copy of your web libraries to your local machine, then add
them to open source Qiqqa. I do this on a local drive and then periodically
backup (from that opening screen) onto my OneDrive. So far no problems.

**Comment by ronved** [comment]
thanks for the quick reply.
does that mean that i have make myself an intranet library and no longer it automatically creates a library with my username?

**Comment by ravindradonde** [comment]
Hi, 
I am Ravindra, my big thank you to you for developing a very nice reference citation qiqqa app.
I am using qiqqa since 2016, at that time qiqqa was working fine, but now when I was shifted to a new and open-source version i.e. setup-v81 and v82.exe.
 I am facing a big problem with BibTeX sniffer tools. The BibTeX sniffer not working correctly, it frequently showing google scholar Recaptcha error. and it not correctly visible Recaptcha in qiqq browser it shows that the browser not supported.

Please suggest to me what should I do?

**Comment by jordanharp** [comment]
Yes, that is correct. Open source Qiqqa does not use usernames or web
libraries.

**Comment by ronved** [comment]
ty.
Is there a mobile version?

**Comment by jordanharp** [comment]
Nope, just Windows for desktop

**Comment by GerHobbelt** [comment]
@ravindradonde : your question has been moved to its own issue: #225. See answer there.

One question or problem per issue, please. Thanks!

**Comment by GerHobbelt** [comment]
@ronved : nope, unfortunately there's no mobile Qiqqa. Quantisle had a mobile app back in the day, but that code has not been open sourced and I have no idea what the mobile version offered, functionality-wise. (As I've been considering X-Platform given #215, I wonder what Qiqqa could do as a mobile app, as keeping a local library like the desktop version is not feasible at least for peeps like me with multi-GB libraries.)

Anyway, resources are limited; mobile is being kept in mind but is currently not the top priority, unless someone else comes in and does #2 + #7 and friends for us first. 😉

**Comment by GerHobbelt** [comment]
> (I previously tried v79 and both account and guest library. I also have web account)

# WARNING

Note this from the qiqqa.com commercial website (emphasis mine):

**After 10 years of your support we have decided to make Qiqqa open source so that it can be grown and extended by its community of thousands of active users.**

***NB: We will be discontinuing Web Library support for Qiqqa at the end of 2020. So you’ll have one year within which to install the latest version of Open Source Qiqqa (which is improving daily), migrate your Web Libraries into Intranet Libraries, and enjoy all the Premium and Premium+ features of Qiqqa for free (except Web Libraries)!***

## Web Libraries (Qiqqa *cloud storage*) will be discontinued 'at the end of 2020'.

**The only way to access your REMOTE = CLOUD-STORED libraries is by using Qiiqa v79, as the Cloud access code was never open sourced.** 

The way that Open Source Qiqqa (v80 and later) *appear* to access your Web Libraries is by discovering the **local copies** of those libraries, which Commercial Qiqqa (v79 and older) kept (*manually*) synchronized.

**Therefore, if you *doubt* or *know* your local copy of your Web Library to be *out of sync* with your cloud-based Web Library (because you or others updated/synced that cloud-based library from other machines after your last sync action on this one) you MUST install Commercial Qiqqa v79 ([available here](https://github.com/GerHobbelt/qiqqa-open-source/releases/tag/v79) among other places) to log into your Qiqqa cloud account, synchronize your Web Libraries with your local copy and then re-install Qiqqa Open Source (v82 preferentially).**

Qiqqa versions v79, v80 and v82 can be installed over one another without issue as they are binary compatible re Qiqqa local library files, so there's no expected harm done in installing v79 over v82, particularly if you limit your activity to syncing Web Libraries. 

(By the way: the `setup.exe` will certainly yak about you installing Qiqqa over a 'newer' version. Disregard and continue.)

Then, once you re-installed v82, it will discover your local Web Library copy again (now synchronized) and you should be good to go.

Do note that the *name* of the library as shown in v82 might be [UUID-like](https://en.wikipedia.org/wiki/Universally_unique_identifier#Format) gobbledigook instead of the proper title you gave it back in the day of Commercial Qiqqa and v79. The contents should be available untrammeled though.
